python/setup.py

```diff
@@ -224,46 +225,33 @@ def _run_cmake(self):
                                      .format(self.boost_namespace))
 
             extra_cmake_args = shlex.split(self.extra_cmake_args)
-            if sys.platform != 'win32':
-                cmake_command = (['cmake'] + extra_cmake_args +
-                                 cmake_options + [source])
-
-                print("-- Runnning cmake for pyarrow")
-                self.spawn(cmake_command)
-                print("-- Finished cmake for pyarrow")
-                args = ['make']
-                if os.environ.get('PYARROW_BUILD_VERBOSE', '0') == '1':
-                    args.append('VERBOSE=1')
-
-                if 'PYARROW_PARALLEL' in os.environ:
-                    args.append('-j{0}'.format(os.environ['PYARROW_PARALLEL']))
-                print("-- Running cmake --build for pyarrow")
-                self.spawn(args)
-                print("-- Finished cmake --build for pyarrow")
-            else:
-                if not is_64_bit:
-                    raise RuntimeError('Not supported on 32-bit Windows')
-
-                # Generate the build files
-                cmake_command = (['cmake'] + extra_cmake_args +
-                                 cmake_options + [source])
-
-                print("-- Runnning cmake for pyarrow")
-                self.spawn(cmake_command)
-                print("-- Finished cmake for pyarrow")
-                # Do the build
-                print("-- Running cmake --build for pyarrow")
-                self.spawn(['cmake', '--build', '.', '--config',
-                            self.build_type])
-                print("-- Finished cmake --build for pyarrow")
+
+            if sys.platform == 'win32' and not is_64_bit:
+                raise RuntimeError('Not supported on 32-bit Windows')
+
+            # Generate the build files
+            print("-- Runnning cmake for pyarrow")
+            self.spawn(['cmake'] + extra_cmake_args + cmake_options + [source])
+            print("-- Finished cmake for pyarrow")
+
+            # Do the build
+            build_tool_args = []
+            if os.environ.get('PYARROW_BUILD_VERBOSE', '0') == '1':
+                build_tool_args.append('VERBOSE=1')
+            if os.environ.get('PYARROW_PARALLEL'):
+                build_tool_args.append(
+                    '-j{0}'.format(os.environ['PYARROW_PARALLEL']))
```

Comment:
Do these options work on Windows?

Reply:
Disabled passing them. I guees I need to spin up a virtualbox sooner or later :)